_episodes/02-filedir.md

```diff
@@ -193,12 +193,12 @@ because its name begins with `/`.
 
 Underneath `/Users`,
 we find one directory for each user with an account on Nelle's machine,
-her colleagues the Mummy and Wolfman.  
+her colleagues the `imhotep` and `larry`.  
 
 ![Home Directories](../fig/home-directories.svg)
 
-The Mummy's files are stored in `/Users/imhotep`,
-Wolfman's in `/Users/larry`,
+The user imhotep's files are stored in `/Users/imhotep`,
```

Comment:
Previously, we used monospaced text when refering to `imhotep` and `larry`; do you think we should do that consistently throughout?